src/Arith.jl

```diff
@@ -178,10 +206,22 @@ function Base.show(io::IO, z::CycloFrac)
 			print(io, "($(z.numerator))//($(z.denominator))")
 		end
 	end
+	print(io, exceptions)
```

Comment:
We can merge it, but ideally, I would restructure this:
```suggestion
    if !isempty(z.exceptions)
	    print(io, "\nWith exceptions:")
	    for exception in z.exceptions
			print(io, "\n  $(exception)")
		end
	end
```
Actually it could also be rewritten to use `join(io, z.exceptions, "\n  ")` I guess -- well and even better: make it use `AbstractAlgebra.Indent()` and `pretty(io)` etc. to realize the indentation. But all that could be in a future PR.

Reply:
I'll do that in another PR.